js/components/root.js

```diff
@@ -15,7 +15,7 @@
 		template: '' +
 		'<div class="section" id="caniupdate">' +
 		'	<h2 class="inlineblock">' +
-		'		'+ t('caniupdate', 'Can I update?') + ' {{ canUpdate }}' +
+		'		'+ t('caniupdate', 'Can I upgrade?') + ' {{ canUpdate }}' +
```

Comment:
In Nextcloud we call this update

Reply:
14 upgrades, about the same number for update-upgrades.